src/buf.gen.yaml

```diff
@@ -0,0 +1,8 @@
+version: v1
+plugins:
+  # This will invoke protoc-gen-es and write output to src/gen
+  - plugin: es
+    out: proto
```

Comment:
As mentioned above, should use `out: src` and move this file out of `src` to the root of the repository